framework/src/controller/helpers/validator/index.js

```diff
@@ -15,6 +16,8 @@ const validatorWithDefaults = new Ajv({
 	useDefaults: true,
 	$data: true,
 });
+validatorWithDefaults.addKeyword('env', envKeyword);
+validatorWithDefaults.addKeyword('arg', argKeyword);
```

Comment:
why do we need these why do we need these keywords?

Reply:
I will add an explanation in JSDoc:
```
/**
 * Adds a possibility to validate the app parameters for Lisk Framework provided through:
 * 1. Environment variables (env) like "npm start LISK_NETWORK=test"
 * 2. Command line arguments (arg) like "npm start --network test"
 * These keywords extend Ajv Validator and follow the format:
 * @type {{compile, errors, modifying, valid, metaSchema}}
 */
```